index.html

```diff
@@ -17,15 +17,25 @@
     <div class="Main-container">
         <div class="container">
             <header class="header hoverable">
-                <a>My Logo</a>
-                <nav><img src="/photos/Union.png" alt="Hamburger Icon"></nav>
+                <h1 class="gol"><a>My Logo</a></h1>
+
+                <nav>
+                    <div class="hidden">
+                        <ul class="hidden-ul">
+                            <li class="po"><a href="#">Portoflio</a></li>
+                            <li class="ab"><a href="#">About</a></li>
+                            <li class="co"><a href="#">Contact</a></li>
+                        </ul>
```

Comment:
your navigation links should  have the same style as the ones in the Figma design, so please it would be better if you try to make the same Figma design 
the font ,color and text decoration

- [ ] Yours
![image](https://user-images.githubusercontent.com/61790587/167229349-5ae30797-55b6-49ec-b5be-c31a49591389.png)

- [ ] Figma design 
![image](https://user-images.githubusercontent.com/61790587/167229373-8e5596bf-0dc8-4935-8cc5-33c6befd474d.png)